### ISSUE TITLE
Decreased cardiac index

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6564]
**Preferred term label:** Decreased cardiac index

**Synonyms**

**Definition (free text, please give PubMed ID)**
Cardiac index adjusts cardiac output to body surface area and therefore describes heart function in respect to the individual body size.

**Parent term (use hpo.jax.org/app)**
Abnormality of blood circulation HP:0011028

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
